lib/active_merchant/billing/gateways/cyber_source.rb

```diff
@@ -813,6 +813,7 @@ def add_auth_network_tokenization(xml, payment_method, options)
             xml.tag!('collectionIndicator', DEFAULT_COLLECTION_INDICATOR)
           end
           xml.tag! 'ccAuthService', { 'run' => 'true' } do
+            xml.tag!('networkTokenCryptogram', payment_method.payment_cryptogram) if payment_method.payment_cryptogram
```

Comment:
You mention that Visa requires this but the `ccAuthService` tag is only used for Mastercards. Visa cards have the cryptogram added to the `ccAuthService.cavv` field

Reply:
Hello and TY @aenand! I corrected the description (Visa -> Cybersource). I meant to express that we got feedback, from Cybersource, asking that the network token cryptogram is provided under the ccAuthService.

Reply:
Ah that makes more sense! I'll take a look at this Monday.

Reply:
It still in draft - I am confused by the fact the cryptogram we be provided twice in the same request.

Reply:
Yeah that is strange 🤔 do you have a contact to reach out to at CyberSource?